react/features/base/config/reducer.ts

```diff
@@ -293,6 +293,48 @@ function _translateInterfaceConfig(oldValue: IConfig) {
         }
     }
 
+    // if we have `deeplinking` defined, ignore deprecated values. Otherwise, compose the config.
+    if (!oldValue.deeplinking) {
+        const disabled = Boolean(oldValue.disableDeepLinking || typeof interfaceConfig !== 'object');
+        const deeplinking: DeeplinkingConfig = {
+            desktopAppName: '',
+            hideLogo: false,
+            disabled,
+            showImage: false,
+            android: {} as DeeplinkingMobileConfig,
+            ios: {} as DeeplinkingMobileConfig
+        };
+
+        if (typeof interfaceConfig === 'object') {
+            const mobileDynamicLink = interfaceConfig.MOBILE_DYNAMIC_LINK;
+            const dynamicLink: MobileDynamicLink | undefined = mobileDynamicLink ? {
+                apn: mobileDynamicLink.APN,
+                appCode: mobileDynamicLink.APP_CODE,
+                ibi: mobileDynamicLink.IBI,
+                isi: mobileDynamicLink.ISI,
+                customDomain: mobileDynamicLink.CUSTOM_DOMAIN
+            } : undefined;
+
+            deeplinking.desktopAppName = interfaceConfig.NATIVE_APP_NAME;
```

Comment:
Why don't we group the desktop options like we do the mobile ones per platform?

Reply:
I thought to do it but for mobile I could define a common type, for desktop it does not quite fit the same type.

Reply:
But yeah I could create a new type.

Reply:
Switched to interfaces and refactored.